examples/hardware_test.rs

```diff
@@ -1,21 +1,126 @@
 #![feature(alloc)]
 #![no_std]
-
+/// Hardware regression tests.
+/// Need P0.03 and P0.04 to be connected (on a nrf52-dk).
+#[macro_use]
```

Comment:
Just for consistency: You could remove this line and replace the `console.write(...)` calls by `writeln!(console, ...)` calls.

Reply:
Done.